frontend/src/pages/agency-profile-form/AgencyProfileForm.js

```diff
@@ -468,41 +572,41 @@ class AgencyProfileForm extends Component {
                   options={[
                     {
                       title: "Monday",
-                      selected: data["distributionDay.monday"],
-                      subkey: "monday",
+                      //selected: data.monday,
+                      selected: true,
```

Comment:
Why is this set as `selected: true` vs `selected: data.monday`?

Reply:
Something about our multi-select dropdowns broke (possibly when we removed bootstrap--maybe a CSS thing), so you can't select any of the options. This would make it so you can't submit the form, because at least one selection is required. I admit I was lazy and did this instead of fixing it, because we'll change this dropdown into a checkbox list when we implement the new distribution frequency section, so multi-select dropdowns won't be needed anymore. Should we still keep them anyway? (I think it adds unnecessary complexity to the component)

Reply:
If you'll be updating this component to something different anyways when you work on your next task then this is fine, but if you'll continue to use multi-select dropdown it should be fixed. I would make everything as simple as possible and if thats changing this component to checkbox list that sounds good. As long as this is a temporary fix that will be updated in your next PR I'm ok with it